File: magecraft.js
//🧙 Magecraft University - JavaScript Edition

const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout,
  });
  
  readline.question("Enter a color: ", (color) => {
    readline.question("Enter an animal: ", (animal) => {
      readline.question("Enter a number between 100 and 900: ", (mana) => {
        console.log("\nWelcome to Magecraft University");
        console.log(`Your alias is ${color} ${animal}`);
        console.log(`Your current mana level is ${mana}, but we will polish you so that you reach your maximum potential!`);
        readline.close();
      });
    });
  });
  

File: magecraft.cpp
//🧙 Magecraft University - Python Edition

#include <iostream>
#include <string>

int main() {
    std::string color, animal;
    int mana;

    std::cout << "Enter a color: ";
    std::getline(std::cin, color);

    std::cout << "Enter an animal: ";
    std::getline(std::cin, animal);

    std::cout << "Enter a number between 100 and 900: ";
    std::cin >> mana;

    std::cout << "\nWelcome to Magecraft University" << std::endl;
    std::cout << "Your alias is The " << color << " " << animal << std::endl;
    std::cout << "Your current mana level is " << mana << ", but we will polish you so that you reach your maximum potential!" << std::endl;

    return 0;
}
